src/styles/divider.css

```diff
@@ -0,0 +1,13 @@
+/* divider */
+
+.ui.divider {
+  background-color: var(--neutral-500);
+  margin: 0;
+  flex-shrink: 0;
+  width: 100%;
+  height: 1px;
+}
+
+.ui.divider:not(.vertical):not(.horizontal) {
+  border: none;
+}
```

Comment:
Let's make the `background-color` transparent on the `.horizontal` and `.vertical` variants, as the line seems to be displayed using a background image.

```
.ui.divider.horizontal,
.ui.divider.vertical {
  background-color: transparent;
}
```